### PR TITLE
fix: removed b2c examples and flows

### PR DESCRIPTION
**Pull request by rupinder-syngh**
- Removed b2c examples and flows

### CODE REVIEW

File: api/docs/customizations.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
     * Food preparation ("less spicy", "less salt", etc.);
     * Packaging related instructions (card for "Happy Birthday", etc.);
 
-Please refer to F&B_Pizza example for [this](https://github.com/ONDC-Official/ONDC-RET-Specifications/blob/draft-1.x/api/components/Examples/B2C/on_search/on_search_F%26B_Pizza.yaml)</br>
```
Comment (tanya-ondc) [inline api/docs/customizations.md:8]: Remove this file.

File: api/docs/return_with_reverse_qc.md [hunk 2]
```diff
@@ -12,4 +12,4 @@
 * It is assumed that the LSP assigned for return pickup will have a certain no of pickup attempts and will trigger scenario B after all pickup attempts failed;
```
Comment (tanya-ondc) [inline api/docs/return_with_reverse_qc.md:12]: Remove this file as well

**Review by tanya-ondc — CHANGES_REQUESTED** [review]

File: api/docs/customizations.md (hunk 1 above)
Comment (rupinder-syngh) [reply api/docs/customizations.md:8]: done

File: api/docs/return_with_reverse_qc.md (hunk 2 above)
Comment (rupinder-syngh) [reply api/docs/return_with_reverse_qc.md]: done

**Review by tanya-ondc — APPROVED** [review]

**Pull request merged**